scripts/build-dist.js

```diff
@@ -45,6 +45,8 @@ babelConfig.babelrc = false;
 babelConfig.presets = babelConfig.presets.map((preset) => {
     return preset === 'es2015' ? 'es2015-rollup' : preset;
 });
+babelConfig.plugins = babelConfig.plugins || [];
+babelConfig.plugins.push(['transform-react-remove-prop-types', {'mode': 'wrap'}]);
```

Comment:
We could move it to `.babelrc`, but it doesn't seem like your currently setup for env builds -  and thats beyond the scope of this PR.